services/packagist/packagist-php-version.spec.js

```diff
@@ -1,5 +1,8 @@
-import { expect } from 'chai'
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
 import PackagistPhpVersion from './packagist-php-version.service.js'
+const { expect } = chai
+chai.use(chaiAsPromised)
```

Comment:
Do we actually need to add this here?

Reply:
If we don’t, these test suite did not pass when I checked ;)

Reply:
Interesting.. This was needed to get the test file to run in isolation, right?

I guess the reason this was passing when the file is run as part of the whole suite is that some other test has registered `chai.use(chaiAsPromised)` but it once the plugin is registered _somewhere_ its registered for the full duration of the test run - it doesn't get cleaned up after each test module.

Reply:
> Interesting.. This was needed to get the test file to run in isolation, right?
> 
Yes.

Your explanation sounds plausible :)